crypto/x509/v3_conf.c

```diff
@@ -487,10 +515,15 @@ static X509V3_CONF_METHOD conf_lhash_method = {
     NULL
 };
 
-void X509V3_set_conf_lhash(X509V3_CTX *ctx, LHASH_OF(CONF_VALUE) *lhash)
+int X509V3_set_conf_lhash(X509V3_CTX *ctx, LHASH_OF(CONF_VALUE) *lhash)
```

Comment:
Technically this is an API break, although minor and probably acceptable. However is it really necessary and worth it? I mean the check for NULL ctx is not needed, or is it? Can it be done at the caller?

Reply:
I also think such an API change should be acceptable because existing calls should not break,
but I agree it's not really worth here (and for the two other such changes), 
and to be on the safe side I've reverted changing their return type.